### ISSUE TITLE
Illegal Value for Line using any input mode while WallabyJs || Quokka is running

### ISSUE THREAD

**Issue by kpruehss** [VSCodeVim/Vim#3459]
With the Vim extension active, starting either Quokka or WallabyJS extensions causes the following issue: 
Approximately one second after entering input mode in any way, an error status will display 

`TaskQueue: Error running task. Failed to handle key=
. Illegal value for 'ine'.`

with the value of `key=` being whichever key was last pressed. Dismissing the error drops Vim back into command mode, but any motion commands will send cursor to either top or bottom of page, with vim becoming entirely unresponsive afterwards. Only restarting vscode appears to resolve the issue.

Possibly similar to issue [#3441 ](https://github.com/VSCodeVim/Vim/issues/3441)
This is new with version *1.0.7*
Issue disappears when reverting to version *1.0.6*

**Comment by justinpincar** [comment]
This is driving me nuts too - I've had to turn off Wallaby or VIM extension and it's been a big hit to my productivity. I noticed it only happens on the build though, so I would wonder if it's similar to https://github.com/VSCodeVim/Vim/issues/3462.

**Comment by kpruehss** [comment]
@justinpincar  You can downgrade to version 1.0.6 of the vim extension and use wallaby again. Not ideal perhaps, but its better than not having vim imho.

**Comment by jpoon** [comment]
I can't repro this at all... I followed http://dm.gl/2015/11/26/wallaby-for-visual-studio-code/ to setup a test project and started wallaby. Don't see any errors with vim.

Or, if there are no negative side-effects with vim, you can press that "Suppress Notification" button.

**Comment by justinpincar** [comment]
@kpruehss Need more coffee... can't believe I didn't think of that. Ty.

@jpoon I tried for a while to consistently reproduce this but couldn't figure out what triggers it. However, once it does happen, it stays broken until VSCode is reloaded. One other tidbit - the cursor jumping definitely corresponds with Wallaby's builds and/or test runs, and it seems to happen right when they finish the run and output the results.

So e.g. I'll be coding somewhere in the middle of the file when Wallaby decides I've paused for long enough to run the tests, I keep typing, and as soon as the Wallaby output shows up it almost seems like my keystrokes are reset and the cursor jumps to the top of the page, usually mangling a few lines where I was as well as the first few lines in the file.

If I can get any more useful info I'll add it to the thread, but I'll prob just keep things downgraded for now.
